app/api/probe_dock/metrics_api.rb

```diff
@@ -117,6 +119,51 @@ def current_organization
         result
       end
 
+      get :projectHealth do
+        authorize! :organization, :data
+
+        test_descriptions_rel = TestDescription.joins(:project_version)
+
+        # Retrieve the data from the specified version otherwise take the most recent project version to retrieve the data
+        if params[:projectVersionId].present?
+          project_version = ProjectVersion.where('api_id = ?', params[:projectVersionId]).first
+          test_descriptions_rel = test_descriptions_rel.joins(:project_version).where('project_versions.api_id = ?', params[:projectVersionId])
+        else
+          project_version = ProjectVersion.joins(:project).where('projects.api_id = ?', params[:projectId]).order('created_at DESC').first
+          test_descriptions_rel = test_descriptions_rel.where('project_versions.id = ?', project_version.id)
+        end
+
+        # In this statement, we use nullif which return null if left value is equal to right value. Therefore, we want to
+        # count the opposite of this results as COUNT will not take into account null values. Ex: all passing tests have
+        # value set to true and then we want to remove the failing ones. Then, we want nullif to return null when passing
+        # is false.
+        test_descriptions_rel = test_descriptions_rel.select(
```

Comment:
This variable should not be named `*_rel`, as it does not contain an Active Record relation, but the result of the query.